scripts/container/README.md

```diff
@@ -0,0 +1,7 @@
+# Container Scripts
```

Comment:
the use of container scripts is a new pattern for c2s projects, is the main reason to wait for the ES instance to be available for the tests to run? 

The nearby services API test uses a different approach of a helper method to verify the ES cluster is ready before running a test https://github.com/nhsuk/nearby-services-api/blob/master/test/lib/testUtils.js

Reply:
We used this approach in `gp-finder` before.